backend/views/profile.js

```diff
@@ -101,81 +101,34 @@ exports.deletePhoto = function(profilePhoto) {
 
  
 
-  // PUT
-   var editProfile = function (req, res, next) {
+  /*
```

Comment:
You need to change the function documentation to be auto-generated afterwards
example/**
	*	@function getUsers - Called on GET "/api/users"	
	*	@params {Object} req - Http request
	* @params {Object} res - Http response
	*	@params {Object} next - Next middleware
	*	@returns {JSON} All users in Database as JSON
 	*/